README.md

```diff
@@ -1,2 +1,71 @@
+[![GitHub contributors](https://img.shields.io/github/contributors/GlobalTechTalks/hackathon.svg)](https://github.com/GlobalTechTalks/hackathon/graphs/contributors)
+
+# AbsoluteHack
+AbsoluteHack is a global hackathon uniting thousands of ignited minds. It is initiated by TechTalks for organizing hackathon in Pan India.  
+In the context of a hackathon, the word "hack" is used to describe how multiple technologies can be used together in a new and innovative way. Hence, AbsoluteHack provides an amazing atmosphere to put in all you've got and bring your ideas to fruition.
+
+# Technology Used
+This consists of two parts:
+* FrontEnd:
+    - HTML
+    - Sass
+    - EJS
+    - Bootstrap
+    - JavaScript
+    - Node as build system
+
+
+* Backend:
+    - Django
+    - PostgreSQL
+
+# Contributing to AbsoluteHack
+
+## General Guidelines
+
+* Add assets such as images and other media in assets folder.
+* Follow standard coding practices, naming conventions and the regular document object model (DOM).
+* Make issues on GitHub to propose new features, bugs and then make a PR referencing the same.
+* Make sure you update your REMOTE ORIGIN in case you have forked this repo : [Refer](https://help.github.com/articles/syncing-a-fork/).
+* Once you update the forked repo make sure you rebase the changes and then open the PR. [Refer](http://stackoverflow.com/questions/7244321/how-do-i-update-a-github-forked-repository).
+
+## Submitting Issues
+
+* Before creating a new issue, perform a [cursory search](https://github.com/GlobalTechTalks/hackathon/issues?utf8=%E2%9C%93&q=) in the main repo to see if a similar issue has already been submitted.
+* You can create an issue [here](https://github.com/GlobalTechTalks/hackathon/issues/new). Please include as many details as possible in your report.
+* Issue titles should be descriptive, explaining at the high level what it is about.
+* Please include the version of the dependencies you are using, as well as what operating system you are having a problem on.
+* _Do not open a [pull request](#pull-requests) to resolve an issue without first receiving feedback from a `collaborator` or `owner` and having them agree on a solution forward_.
+* Include screenshots and animated GIFs whenever possible; they are immensely helpful.
+
+
+## Pull Requests
+
+* **DO NOT ISSUE A PULL REQUEST WITHOUT FIRST [SUBMITTING AN ISSUE](#submitting-issues).**
+* It is now mandatory for everyone to comment on every PR made and read other people's code.
+* Pull requests should reference their related issues. If the pull request closes an issue, [please reference its closing in your commit messages](https://help.github.com/articles/closing-issues-via-commit-messages/). Pull requests not referencing any issues will be closed.
+* Pull request titles should be descriptive, explaining at the high level what it is doing, and should be written in the same style as [git commit messages](#git-commit-messages).
```

Comment:
@pythongiant Please follow this guideline and change your PR title accordingly

Reply:
Also, follow [this](https://help.github.com/articles/closing-issues-using-keywords/) to close the issue after PR is merged.

Reply:
done